### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by dpeterson8**
Still working on screen responsive, landing card style, and plays endpoint and display page.
Site link: https://proud-smoke-0a4e0fe1e.3.azurestaticapps.net/

### CODE REVIEW

**Comment by DominicAlvarez** [comment]
Make all pages responsive ❌
1080p desktop ✅
iPad Air ❌
Samsung Galaxy S20 Ultra Super Mega Excellent ❌
IPhone SE ❌
Create a improved landing page and site theme (Background, make it look attractive to play) ✅
Theme should extend all pages ❌
Add an instructions page that is available via the hamburger menu that details the features of the game and how to play ✅
Create a new Wordle game page that reuses components but will provide the same word for a specific day. Everyone going to the page that day will get the same word. The page should indicate Wordle of the Day or something like that ✅
Create a page that lists the last 10 daily words with statistics. Number of plays, Average score, Average time. (just list the date, don't show the word silly) ❌
It should indicate if the user has played that day ❌
Clicking on the item should take the player to that game (Extra Credit) ✅

The landing page created is very nice and i enjoy that there is the 3 buttons to take a user to play, leaderboard or instructions. Project looks good and im sure your focus will be shifting from the backend stuff to make sure word of the day works to the designs. Missing the stats for the word of the day. Looks good otherwise!

**Review by mmwoodfo — COMMENTED** [review]
A couple of issues with responsive design, some of the dialogs and menus are a bit crowded
image

image

File: wordle-web/src/components/TutorialCard.vue [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<template>
```
Comment (mmwoodfo) [inline wordle-web/src/components/TutorialCard.vue:1]: This looks great!

File: wordle-web/src/views/WordleView.vue [hunk 2]
```diff
@@ -85,13 +90,25 @@ onUnmounted(() => {
 
 function newGame() {
   overlay.value = true
-  Axios.get('word')
+  let apiPath = 'word'
+  if (route.path == '/wordoftheday') {
+    apiPath = `word/wordoftheday?offsetInHours=${new Date().getTimezoneOffset() / -60}`
+    if (route.query.date) {
+      apiPath += `&date=${route.query.date}`
```
Comment (mmwoodfo) [inline wordle-web/src/views/WordleView.vue:97]: I see you're trying to grab the date from the router path here, but I don't think you ever set up the route to accept route parameters? Are you able to actually hit this page with a date and have this work?

**Pull request closed without merging**